### PR TITLE
Refine _blank implicit noopener

### PR DESCRIPTION
**Pull request by kiding**
Some tags with their target to _blank have noopener by default. As mentioned in https://github.com/mdn/browser-compat-data/pull/7642, this behavior is now part of the standard.

This PR removes the vendor/version-specific wordings, and adds links to the browser compatibility. The idea is that those who stumble upon the `Window.opener` document could easily end up at the corresponding compatibility data.

### CODE REVIEW

File: files/en-us/web/html/element/a/index.html [hunk 1]
```diff
@@ -94,7 +94,7 @@ <h2 id="Attributes">Attributes</h2>
  </div>
 
  <div class="note">
- <p><strong>Note:</strong> In newer browser versions (e.g. Firefox 79+) setting <code>target="_blank"</code> on <code>&lt;a&gt;</code> elements implicitly provides the same <code>rel</code> behavior as setting <code>rel="noopener"</code>.</p>
+ <p><strong>Note:</strong> Setting <code>target="_blank"</code> on <code>&lt;a&gt;</code> elements implicitly provides the same <code>rel</code> behavior as setting <code>rel="noopener"</code>. See <a href="#Browser_compatibility">browser compatibility</a> for support status.</p>
```
Comment (rachelandrew) [inline files/en-us/web/html/element/a/index.html:97]: While here can you update this to use the new notecard style?

https://developer.mozilla.org/en-US/docs/MDN/Guidelines/CSS_style_guide#.note.notecard

**Review by rachelandrew — CHANGES_REQUESTED** [review]
the way we style notes has changed, so if you could update that while in here that would be fantastic.

File: files/en-us/web/html/element/a/index.html (hunk 1 above)
Comment (kiding) [reply files/en-us/web/html/element/a/index.html]: Done! → b4a5f8f9a285511e4b09d6e5778018cae84f4a5e

**Review by rachelandrew — APPROVED** [review]
thanks :)

**Pull request merged**